numpy/lib/nanfunctions.py

```diff
@@ -508,7 +515,7 @@ def nansum(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
 
     Numpy integer arithmetic is modular. If the size of a sum exceeds the
     size of an integer accumulator, its value will wrap around and the
-    result will be incorrect. Specifying ``dtype=double`` can alleviate
+    result will be incorrect. Specifying ``dtype=np.double`` can alleviate
```

Comment:
If you're going to keep this language, don't keep the word "double" which is non-standard from a numpy perspective. "float" or "float64" would be the more standard terms.